src/seeThru.js

```diff
@@ -249,22 +249,25 @@
 		, requestAnimationFrame = window.requestAnimationFrame || getRequestAnimationFrame()
 		, cancelAnimationFrame = window.cancelAnimationFrame || getCancelAnimationFrame()
 		, drawFrame = function(recurse){
+			var image, alphaData, i, len, currentTime = video.currentTime;
+			
+			if (lastFrameTime !== currentTime) {

```

Comment:
From what I understand `lastFrameTime` is never being declared anywhere, thus making it a global. It'd need to be declared on the outer scope of the function.